addons/core/functions/server/fnc_serverInit.sqf

```diff
@@ -113,6 +113,17 @@ GVAR(instanciationIsReady) = false;
             VARIABLE_DEFAULT(TFAR_freq_sr_west,true call DFUNC(generateSRSettings));
             VARIABLE_DEFAULT(TFAR_freq_sr_east,true call DFUNC(generateSRSettings));
             VARIABLE_DEFAULT(TFAR_freq_sr_independent,true call DFUNC(generateSRSettings));
+
+            if (TFAR_SameSRFrequenciesPerGroup) then {
+                {
+                    _x params ["_unit"];
+                    private _frequencies = _unit getVariable ["TFAR_freq_sr", (group _unit) getVariable "TFAR_freq_sr"];
+                    if (isNil "_frequencies") then {
+                        _frequencies = [TFAR_MAX_CHANNELS, TFAR_MAX_SW_FREQ, TFAR_MIN_SW_FREQ, TFAR_FREQ_ROUND_POWER] call TFAR_fnc_generateFrequencies;
+                        (group _unit) setVariable ["TFAR_freq_sr", _frequencies, true];
+                    };
+                } forEach (playableUnits);
```

Comment:
```suggestion
                } forEach allPlayers;
```

Reply:
Maybe this should be independent of "TFAR_SameSRFrequenciesForSide".
Otherwise it taking no effect if SameSideFreq's are also enabled is badly documented atleast.

Reply:
Oops, actually this won't work. serverInit runs once. Players might join JIP and their groups then won't get preset frequencies as they only get created when the player joins.

Reply:
> Maybe this should be independent of "TFAR_SameSRFrequenciesForSide".

Only one of them can be used at a time. Yes it can confuse.
As you can see in first commit I did it independent of "TFAR_SameSRFrequenciesForSide".

> Oops, actually this won't work.

😞

Reply:
>Oops, actually this won't work.

We need some kind of player joined/group created eventhandler. Definitely possible.

Reply:
2.10 will add group eventhandlers, including join.